doc/changelog/11-infrastructure-and-dependencies/11131-ci+back_to_ocaml_410.rst

```diff
@@ -0,0 +1,4 @@
+- **Added:**
+  Bump official OCaml support and CI testing to 4.10.0
+  (`#11131 <https://github.com/coq/coq/pull/11131>`_,
+  by Emilio Jesus Gallego Arias).
```

Comment:
Maybe mention the people that contributed the fixes for 4.10 compatibility and the relevant PRs as well.

Reply:
They didn't have their own changelog? I'll have a look tho. Also, this reminds me I should close some issues likely.

Reply:
No, they didn't. I think it makes sense to explain the whole bunk of change in just a single entry though.

Reply:
Sounds good, will add an overview.